community/pyqt5/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+python configure.py \
+    --confirm-license
+
+# Build single threaded to avoid racecondition
```

Comment:
This only *installs* as single threaded no? The build still uses `MAKEFLAGS` right?

Reply:
Oh yes. The buildfile instructions are fine. The comment so lala.